contentcuration/contentcuration/frontend/shared/views/Icon.vue

```diff
@@ -1,45 +1,37 @@
-<script>
+<template>
 
-  import isArray from 'lodash/isArray';
-  import isObject from 'lodash/isObject';
-  import { VIcon } from 'vuetify/lib/components/VIcon';
+  <KIcon
+    v-bind="$attrs"
+    :icon="icon"
+    :color="color"
+    :class="{ 'rtl-flip': rtlFlip }"
```

Comment:
I'm guessing you were consolidating previous functionality here, which is great! Although it seems `KIcon` already has handling for flipping icons for RTL languages. So perhaps this can be removed? I don't actually see anything using the `rtlFlip` prop on this component

Reply:
Yup. Removed it already. Thanks for pointing out!